src/components/TextField/components/Resizer/tests/Resizer.test.tsx

```diff
@@ -58,52 +53,47 @@ describe('<Resizer />', () => {
   describe('contents', () => {
     it('renders contents', () => {
       const contents = 'Contents';
-      const resizer = mountWithAppProvider(
+      const resizer = mountWithApp(
         <Resizer {...mockProps} contents={contents} />,
       );
-      const contentsNode = findByTestID(resizer, 'ContentsNode');
-      expect(contentsNode.text()).toBe(contents);
+      expect(resizer.find('div')).toContainReactText(contents);
     });
 
     it('encodes HTML entities', () => {
       const contents = `<div>&\nContents</div>`;
-      const resizer = mountWithAppProvider(
+      const resizer = mountWithApp(
         <Resizer {...mockProps} contents={contents} />,
       );
-      const contentsNode = findByTestID(resizer, 'ContentsNode');
       const expectedEncodedContents =
         '&lt;div&gt;&amp;<br>Contents&lt;/div&gt;<br></div>';
-      expect(contentsNode.html()).toContain(expectedEncodedContents);
+      expect(resizer.html()).toContain(expectedEncodedContents);
     });
 
     it('ignores carriage returns when rendering content', () => {
       const contents = `<div>&\n\r\r\rContents</div>`;
-      const resizer = mountWithAppProvider(
+      const resizer = mountWithApp(
         <Resizer {...mockProps} contents={contents} />,
       );
-      const contentsNode = findByTestID(resizer, 'ContentsNode');
       const expectedEncodedContents =
         '&lt;div&gt;&amp;<br>Contents&lt;/div&gt;<br></div>';
-      expect(contentsNode.html()).toContain(expectedEncodedContents);
+      expect(resizer.html()).toContain(expectedEncodedContents);
     });
   });
 
   describe('minimumLines', () => {
     it('renders a number of <br> tags equivalent to minimumLines', () => {
       const minimumLines = 3;
-      const resizer = mountWithAppProvider(
+      const resizer = mountWithApp(
         <Resizer {...mockProps} minimumLines={minimumLines} />,
       );
-      const breakingSpaces = findByTestID(resizer, 'MinimumLines');
-      expect(breakingSpaces.html()).toContain('<br><br><br>');
+      expect(resizer.find('div')!.html()).toContain('<br><br><br>');
```

Comment:
This part is where I have more doubts, not sure if someone can have a suggestion that is better related to this.

Reply:
I think here would be better to use toContainReactHTML. I will check if that is the correct approach.

Reply:
```suggestion
     expect(resizer.find('div'))!. toContainReactHTML('<br><br><br>');
```
Would have been my thoughts on this.

Reply:
My bad, `toContainReactHtml` not `toContainReactHTML`:

```js
expect(resizer.find('div'))!.toContainReactHtml('<br><br><br>');
```

Reply:
don't worry! I push it but i need to add it to the other test in the list.